pipeline/terraform/modules/stack/service_work_batcher.tf

```diff
@@ -52,12 +52,17 @@ module "batcher_lambda" {
     max_batch_size = 40
   }
 
+  timeout = 60 * 5 # 5 Minutes
```

Comment:
I think the lambda is already shutting down in the midst of processing with a 10 minute timeout